charmcraft.yaml

```diff
@@ -4,6 +4,9 @@ parts:
   charm:
     charm-python-packages: [setuptools < 58]
 
+links:
+  documentation: https://discourse.charmhub.io/t/charm-nginx-doc/15015
+
```

Comment:
metadata.yaml exists, so you must put links there.  We can only use the new unified charmcraft.yaml spec if metadata.yaml is not used at all.

```
$ charmcraft -v pack
Starting charmcraft version 2.7.1
Logging execution to '/home/ubuntu/.local/state/charmcraft/log/charmcraft-20240807-083011.743864.log'
Cannot specify 'links' in charmcraft.yaml when 'metadata.yaml' exists
```

Reply:
Removed metadata to mirror https://github.com/canonical/charm-cloudsupport/pull/28/files